From 595455061248a4bfe054abd6754f35f60949a030 Mon Sep 17 00:00:00 2001
From: Paulo Lieuthier <paulolieuthier@gmail.com>
Date: Fri, 17 Oct 2014 21:24:27 -0300
Subject: [PATCH] Remove plugins' margins from themes files

So the user can use que panel edges to activate plugins.
---
 themes/a-mego/lxqt-panel.qss            | 2 +-
 themes/ambiance/lxqt-panel.qss          | 2 +-
 themes/flat-dark-alpha/lxqt-panel.qss   | 2 +-
 themes/green/lxqt-panel.qss             | 2 +-
 themes/light/lxqt-panel.qss             | 2 +-
 themes/plasma-next-alpha/lxqt-panel.qss | 2 +-
 6 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/themes/a-mego/lxqt-panel.qss b/themes/a-mego/lxqt-panel.qss
index 9c62ecf..c027f1f 100644
--- a/themes/a-mego/lxqt-panel.qss
+++ b/themes/a-mego/lxqt-panel.qss
@@ -20,7 +20,7 @@ LxQtPanel[position="Top"] #BackgroundWidget {
  * General plugins settings
  */
 Plugin {
-    margin: 3px 0 3px 0;
+    margin: 0;
     padding: 0;
     spacing: 20px;
 }
diff --git a/themes/ambiance/lxqt-panel.qss b/themes/ambiance/lxqt-panel.qss
index d102754..8877ec8 100644
--- a/themes/ambiance/lxqt-panel.qss
+++ b/themes/ambiance/lxqt-panel.qss
@@ -22,7 +22,7 @@ LxQtPanel[position="Top"] #BackgroundWidget {
  */
 Plugin, 
 LxQtPanelPlugin {
-    margin: 2px 0 2px 0;
+    margin: 0;
     padding: 0;
     spacing: 20px;
     qproperty-moveMarkerColor: #C94C21;
diff --git a/themes/flat-dark-alpha/lxqt-panel.qss b/themes/flat-dark-alpha/lxqt-panel.qss
index 2fb4cfc..6452e24 100644
--- a/themes/flat-dark-alpha/lxqt-panel.qss
+++ b/themes/flat-dark-alpha/lxqt-panel.qss
@@ -78,7 +78,7 @@ VolumePopup  > QSlider::sub-page:vertical {
 
 Plugin,
 LxQtPanelPlugin {
-    margin: 2px 0 2px 0;
+    margin: 0;
     padding: 0;
     spacing: 20px;
     qproperty-moveMarkerColor: #C94C21;
diff --git a/themes/green/lxqt-panel.qss b/themes/green/lxqt-panel.qss
index 51dacf0..553d007 100644
--- a/themes/green/lxqt-panel.qss
+++ b/themes/green/lxqt-panel.qss
@@ -29,7 +29,7 @@ LxQtPanel[position="Right"] #BackgroundWidget {
  * General plugins settings
  */
 Plugin {
-    padding: 0 2px 0 2px;
+    padding: 0;
     spacing: 20px;
 }
 
diff --git a/themes/light/lxqt-panel.qss b/themes/light/lxqt-panel.qss
index 73c3b63..5462680 100644
--- a/themes/light/lxqt-panel.qss
+++ b/themes/light/lxqt-panel.qss
@@ -29,7 +29,7 @@ LxQtPanel[position="Right"] #BackgroundWidget {
  * General plugins settings
  */
 Plugin {
-    padding: 0 2px 0 2px;
+    padding: 0;
     spacing: 20px;
 }
 
diff --git a/themes/plasma-next-alpha/lxqt-panel.qss b/themes/plasma-next-alpha/lxqt-panel.qss
index 8415022..f039b22 100644
--- a/themes/plasma-next-alpha/lxqt-panel.qss
+++ b/themes/plasma-next-alpha/lxqt-panel.qss
@@ -75,7 +75,7 @@ VolumePopup  > QSlider::sub-page:vertical {
 
 Plugin,
 LxQtPanelPlugin {
-    margin: 2px 0 2px 0;
+    margin: 0;
     padding: 0;
     spacing: 20px;
     qproperty-moveMarkerColor: #C94C21;
